src/proof/alf/alf_printer.cpp

```diff
@@ -188,6 +195,18 @@ bool AlfPrinter::isHandled(const ProofNode* pfn) const
   return false;
 }
 
+bool AlfPrinter::isHandledTheoryRewrite(ProofRewriteRule id,
+                                        const Node& n) const
+{
+  switch (id)
+  {
+    case ProofRewriteRule::DISTINCT_ELIM:
+    case ProofRewriteRule::RE_LOOP_ELIM: return true;
```

Comment:
Do we have the Alf code to handle this rule?

Reply:
Yes, it was added in https://github.com/cvc5/cvc5/pull/10685.